includes/header.php

```diff
@@ -77,10 +77,11 @@
   // Add ID attributes to headers
   $hids = Array();
   $content = preg_replace_callback(
-    '~<h([1234])>([^<]*)</h([1234])>~Ui', // Ungreedy by default, case insensitive
+    '~<h([1234])>(\S*)</h([1234])>~Ui', // Ungreedy by default, case insensitive
```

Comment:
Does this work with headers that have whitespace? eg. `<h1>This Header</h1>`?

More generally, why the change?

Reply:
I had to change it, because ```[^<]*``` didn't catch ```<code>match</code>``` pieces.
Hmm, yes, this actually breaks headers with a whitespace, sorry. 
How about ```~<h([1234])>(.*?)</h([1234])>~Ui``` or is that to general?

Reply:
Yes I was going to say that it should work, as long as we have the ungreedy `U` modifier so that it doesn't match the whole file...

Reply:
I made it more general now (see 164ceee). 
While testing, I found two different problems with the links from the ToC of the Usage section of bcellmagic (https://nf-co.re/bcellmagic/docs/usage). 
- The `-profile`'s href is to `#-profile-single-dash` 
= The `<li>`s underneath shouldn't appear there, since they are no `<h>` elements. Not sure if that is coming from the repo or the backend, so I don't know where to open the respective issue or fix the problem.

Reply:
`-profile` should be one hyphen (it’s a core Nextflow option, not a pipeline param) and the markdown (including ToC) comes from the pipeline. So needs to be fixed there if anywhere I guess..